unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -31,7 +31,7 @@ export default function walletMiddleware({ getState, dispatch }) {
   // Buffer of actions waiting for connection
   const actions = []
 
-  const walletService = new WalletService()
+  const walletService = new WalletService(config.providers, config.isServer)
```

Comment:
Great!
I was actually thinking about your complaints that we inject the configuration everywhere and I was wondering if there was just a way to pass the *full* config object by default everywhere and then use named assignments in the constructor signatures... but that's for another PR!

Reply:
Yes this would be a great solution to that particular issue. Would you like that PR to be a follow-up or a precursor to this one?

Reply:
I'll do it as a precursor because why not :)